### DISCUSSION TITLE
2.0 按照报错了，大佬们救救孩子吧ERROR: Encountered errors while bringing up the project.

### DISCUSSION THREAD

**Discussion by Jeffreyone** [halo-dev/halo#2823] [Q&A]
<img width="1114" alt="uTools_1669928067521" src="https://user-images.githubusercontent.com/49830396/205157454-24cadbe9-e558-444f-a06b-93b8da189a21.png">
配置完全按照指南操作的，第一次都正常安装启动了，但是不能通过本地 IP 访问，我就删除文件卸载容器，重新操作就出现了这个问题，求助。。

**Comment by JohnNiang** [comment]
试着执行以下命令看看具体发生了什么：

```bash
docker-compose ps -a
docker-compose logs
```

**Reply by Jeffreyone** [reply]
我重新运行了 docker-compose up 后正常了，没有报错了。但是页面打开提示：localhost 拒绝了我们的连接请求。

**Comment by Jeffreyone** [comment]
![](obsidian://open?vault=%E7%9F%A5%E8%AF%86%E5%82%A8%E5%A4%87&file=%E5%9B%BE%E5%BA%93%2FPasted%20image%2020221202144853.png)

![](obsidian://open?vault=%E7%9F%A5%E8%AF%86%E5%82%A8%E5%A4%87&file=%E5%9B%BE%E5%BA%93%2FPasted%20image%2020221202144853.png)
@JohnNiang

**Comment by Jeffreyone** [comment]
<img width="1445" alt="uTools_1669964074252" src="https://user-images.githubusercontent.com/49830396/205233579-986b5b4e-c827-4056-b726-fe5334a210e9.png">

**Reply by Jeffreyone** [reply]
只执行到数据库，halo 那部分就进行不下去了

**Reply by JohnNiang** [reply]
> 只执行到数据库，halo 那部分就进行不下去了

看样子是 MySQL 没起来，导致 Halo 无法正常启动。

试试 PostgreSQL 呢

**Reply by Jeffreyone** [reply]
我重新运行PostgreSQL也是报错，但是我再次运行 docker-compose up 这条命令之后正常了，没有报错。但是页面打开提示：localhost 拒绝了我们的连接请求。

**Reply by Jeffreyone** [reply]
[halo运行日志.txt](https://github.com/halo-dev/halo/files/10139910/halo.txt)

**Reply by Jeffreyone** [reply]
我似乎找到问题了，就是 yaml 里面没有改地址。。。抱歉
